### PR TITLE
Remove build type from package name

### PR DESCRIPTION
**Pull request by MariaSemple**
### Intent

Rollback the change that added the build type to the package name.

### Approach

Remove the type of build from the package name.

### Automated Tests

Automated tests rely on package name to install RStudio.

### QA Notes

This impacts fuzzbucket builds.

### Checklist

- [x] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [x] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [x] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [x] This PR passes all local unit tests

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -369,15 +369,8 @@ try {
...
-                  script: "@type BUILDTYPE",
-                  returnStdout: true
-                ).trim().toLowerCase()
-
-                def packageName = "RStudio-${buildType}-${rstudioVersionMajor}.${rstudioVersionMinor}.${rstudioVersionPatch}-RelWithDebInfo"
-                if (buildType == "release") {
-                  packageName = "RStudio-${rstudioVersionMajor}.${rstudioVersionMinor}.${rstudioVersionPatch}-RelWithDebInfo"
-                }
+
+                def packageName = "RStudio-${rstudioVersionMajor}.${rstudioVersionMinor}.${rstudioVersionPatch}"
```
Comment (jmcphers) [inline Jenkinsfile:373]: Doesn't this still need RelWithDebInfo since that's the filename produced by cpack? (we don't rename it until upload)

Comment (MariaSemple) [inline Jenkinsfile:373]:
```suggestion
                def packageName = "RStudio-${rstudioVersionMajor}.${rstudioVersionMinor}.${rstudioVersionPatch}-RelWithDebInfo"
```

Comment (MariaSemple) [reply Jenkinsfile]: Fixed

**Review by jmcphers — APPROVED** [review]

**Review by R-Hannibal — APPROVED** [review]
LGTM 👍.

**Pull request merged**